BridgeAppSDK/SBAActiveTask.swift

```diff
@@ -44,6 +44,7 @@ public enum SBAActiveTaskType {
     case walking
     case tremor
     case moodSurvey
+    case trailmaking
```

Comment:
Would it be useful instead to use this an example of adding a custom task at the app level? Or is it more useful to have it built in to the SDK do you think?